keyboards/melody96/keymaps/areyouconfused/melody96.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
Really not a fan of this addition, as its got a lot of potential to cause issues (and not a paradigm that would be clean if it propagates through the repo). It relies on replacing an existing implementation based on QMKs lax use of VPATH, which would need to be kept in step with any renames and such.